distrobuilder/main.go

```diff
@@ -829,6 +829,19 @@ fi
 if [ "${nm_exists}" -eq 1 ]; then
 	fix_nm_link_state eth0
 fi
+
+# Allow masking units created by the lxc system-generator.
+for d in /etc/systemd/system /usr/lib/systemd/system /lib/systemd/system; do
+	if ! [ -d "${d}" ]; then
+		continue
+	fi
+
+	find "${d}" -maxdepth 1 ! -type d | while read f; do
```

Comment:
Would it work to just ask for symlinks with `find`'s `-type l`? Either way, `read -r` should be used to disable escape char handling.

Reply:
You're right, the masked units are all symlinks. Didn't think about that. I'll fix both. Thanks!